egs/aishell/s10/chain/tdnnf_layer.py

```diff
@@ -213,10 +206,10 @@ def forward(self, x):
 
         # TODO(fangjun): implement GeneralDropoutComponent in PyTorch
 
-        if self.linear.kernel_size == 3:
-            x = self.bypass_scale * input_x[:, :, 1:-1:self.conv_stride] + x
+        if self.linear.kernel_size == 2:
+            x = self.bypass_scale * input_x[:, :, self.subsampling_factor:-self.subsampling_factor:self.subsampling_factor] + x
```

Comment:
to make this line easier to read I'd use a variable `s = self.subsampling_factor`, defined just before this.

Reply:
OK